### DISCUSSION TITLE
Checkbox checked hover effect

### DISCUSSION THREAD

**Discussion by Saicharan0662** [ant-design/ant-design#39479] [Q&A]
I am getting blue border when hover on checkbox and label while checkbox is checked (not getting when it's not checked). 
I tried few things but no luck. The code is attached.

`.ant-checkbox-wrapper-checked:hover .ant-checkbox-inner,
.ant-checkbox-checked:hover .ant-checkbox-inner {
  border-color: rgba(40, 40, 40, 0.8) !important;
}`

`.ant-checkbox-wrapper:hover .ant-checkbox-inner,
.ant-checkbox:hover .ant-checkbox-inner,
.ant-checkbox-input:focus+.ant-checkbox-inner {
  border-color: #d9d9d9 !important;
}`

Thanks in advance!!

**Comment by khjabir** [comment]
`.ant-checkbox-checked:after { border: none !important }`

**Reply by paaulorobson** [reply]
Is it possible to remove it using a-config-provider?
